index.html

```diff
@@ -288,6 +290,53 @@ <h5>LET'S CONNECT</h5>
 
     </section>
 
+    <section class="contact_section">
+
+        <div class="contact_form">
+
+            <header>
+                <h3>Contact me</h3>
+            </header>
+
+            <p>
```

Comment:
- [ ] **[OPTIONAL]:** Sorry that this wasn't pointed out by a previous reviewer but remember that the main goal of these activities is to follow as close as possible a given design. 🎨 At the moment, your contact section is missing some background decorations: 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/199486685-75227e0f-2d77-4589-b3b8-de4bf10ffb0f.png) | ![imageedit_3_4641310007](https://user-images.githubusercontent.com/77069011/199486583-72e62eb8-1fe9-4df2-ab89-b533369a0e8c.gif) |


- [ ] **[OPTIONAL]:** It would also be a great idea to fix the size of the `<textarea>`.

Please, consider fixing this. 🛠️